### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by juanudl96**

### CODE REVIEW

**Comment by juanudl96** [comment]
For pulling new commints

File: src/main/java/cat/udl/eps/softarch/learnphysics/repository/ExamRepository.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package cat.udl.eps.softarch.learnphysics.repository;
+
+public interface ExamRepository {
```
Comment (bergacat1) [inline src/main/java/cat/udl/eps/softarch/learnphysics/repository/ExamRepository.java]: You should add the annotations and extend the corresponding class in order that this class exposes the http endpoints. It is not doing anything right now.

**Review by bergacat1 — CHANGES_REQUESTED** [review]

**Review by bergacat1 — APPROVED** [review]

File: src/main/java/cat/udl/eps/softarch/learnphysics/domain/Question.java [hunk 2]
```diff
@@ -18,7 +19,9 @@
 public class Question {
```
Comment (bergacat1) [inline src/main/java/cat/udl/eps/softarch/learnphysics/domain/Question.java:7]: Without the annotation @Entity Spring does not know this is a class of the model and you can not create relations with other classes. This is causing the application to crash.

**Review by bergacat1 — DISMISSED** [review]

File: src/main/java/cat/udl/eps/softarch/learnphysics/domain/Question.java (hunk 2 above)
Comment (bergacat1) [reply src/main/java/cat/udl/eps/softarch/learnphysics/domain/Question.java:7]: Check that the annotation is present in all classes of the model.

**Review by rogargon — APPROVED** [review]

**Pull request merged**